app/core/views.py

```diff
@@ -0,0 +1,174 @@
+import random
+import logging
+from uuid import UUID
+
+from rest_framework import viewsets
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from django.core.serializers import serialize
+from rest_framework import status
+from rest_framework.permissions import IsAdminUser
+from rest_framework import mixins
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import connection
+from django.contrib.auth import get_user_model
+
+from .permissions import PlayerPermission
+
+from .models import (
+    CategoryQuestionModel, AnswerModel, CategoryModel, QuestionModel,
+    RankingModel
+)
+from .serializers import (
+    CategoryQuestionSerializer, AnswerSerializer, CategorySerializer,
+    QuestionSerializer, QuizSerializer, RankingSerializer, UserSerializer
+)
+
+class UserViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
+    UserModel = get_user_model()
+    queryset = UserModel.objects.all()
+    serializer_class = UserSerializer
+
+
+class CategoryViewSet(viewsets.ModelViewSet):
+    queryset = CategoryModel.objects.all()
+    serializer_class = CategorySerializer
+    permission_classes = (
+        PlayerPermission,
+    )
+
+class QuestionViewSet(viewsets.ModelViewSet):
+    queryset = QuestionModel.objects.all()
+    serializer_class = QuestionSerializer
+    permission_classes = [IsAdminUser]
+
+class AnswerViewSet(viewsets.ModelViewSet):
+    queryset = AnswerModel.objects.all()
+    serializer_class = AnswerSerializer
+    permission_classes = [IsAdminUser]
+
+class CategoryQuestionViewSet(viewsets.ModelViewSet):
+    queryset = CategoryQuestionModel.objects.all()
+    serializer_class = CategoryQuestionSerializer
+    permission_classes = [IsAdminUser]
+
+
+class RankingViewSet(
+    mixins.ListModelMixin, mixins.RetrieveModelMixin, viewsets.GenericViewSet
+):
+    queryset = RankingModel.objects.all().order_by('-value')
+    serializer_class = RankingSerializer
+    permission_classes = [PlayerPermission]
+    
+    # Neste caso, pk eh o id da category.
+    def retrieve(self, request, pk):
```

Comment:
Você pode nomear o parâmetro da forma que quiser. Nesse caso, se ele representa o id da categoria seria mais interessante chamá-lo de `category_id`.

Reply:
Sim, perfeitamente. Concordo com você. Seria até muito mais legível. Faltou o Zen do Python....Explicit is better than implicit.